src/ConnectedMode.UnitTests/Persistence/LegacySolutionBindingPathProviderTests.cs

```diff
@@ -49,21 +49,21 @@ public void Ctor_DoesNotCallServices()
 
         [TestMethod]
         [DataRow(null, null)] // null solution -> solution not open -> return null
-        [DataRow(@"c:\test\any.sln", @"c:\test\SonarQube\SolutionBinding.sqconfig")]
-        [DataRow(@"c:\aaa\bbb\x.y.x.any.sln", @"c:\aaa\bbb\SonarQube\SolutionBinding.sqconfig")]
-        public void Get_ReturnsExpectedValue(string solutionPathToReturn, string expectedResult)
+        [DataRow(@"c:\test", @"c:\test\SonarQube\SolutionBinding.sqconfig")]
+        [DataRow(@"c:\aaa\bbb", @"c:\aaa\bbb\SonarQube\SolutionBinding.sqconfig")]
+        public void Get_ReturnsExpectedValue(string solutionDirectoryToReturn, string expectedResult)
```

Comment:
Fixed up the existing test.
Previously, the product code called "get solution file name" and used `Path.GetDirectoryName` to calculate the directory. Now it just calls the "get directory" method.